vnext/ReactWindowsCore/tracing/tracing.cpp

```diff
@@ -33,6 +33,7 @@ std::mutex g_pages_mutex;
 }
 
 /*static */ void FbSystraceAsyncFlow::end(uint64_t tag, const char *name, int cookie) {
+  std::lock_guard<std::mutex> guard(s_tracker_mutex_);
```

Comment:
>std::lock_guard<std::mutex> guard(s_tracker_mutex_); [](start = 2, length = 52)

The 'new fashion' is to write it as:

  std::scoped_lock lock{s_tracker_mutex_};

Here we use the new C++17 scoped_lock class and the C++ 17 ability to deduce the argument type in constructors.